package.json

```diff
@@ -34,11 +34,13 @@
     "@nomiclabs/hardhat-web3": "^2.0.0",
     "@openzeppelin/contracts": "^4.5.0",
     "@openzeppelin/contracts-upgradeable": "^4.5.1",
+    "chai": "^4.3.6",
```

Comment:
I don't think chai needs to be added to dependencies right? It's fine in `devDependencies`.